tests/history-missing-bug.rs

```diff
@@ -0,0 +1,168 @@
+pub mod utils;
+
+use rgb::{interface::{AmountChange, IfaceOp}, XOutputSeal, XWitnessId};
```

Comment:
imports should go to `tests/utils/mod.rs`, also I don't think we need a dedicated file for this, please move the tests to `tests/transfers.rs`